app/domain/operations/hbx_admin/dry_run/individual/analyzer.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+
+module Operations
+  module HbxAdmin
+    module DryRun
+      module Individual
+        # This Operation is responsible for getting the renewal dry run count for applications, notices and benefits.
+        class Analyzer
+          include Dry::Monads[:result, :do]
+
+          APPLICATION_STATES = %w[
+            renewal_draft submitted applicants_update_required income_verification_extension_required
+            haven_magi_medicaid_eligibility_request_errored mitc_magi_medicaid_eligibility_request_errored
+            submission_pending determined determination_response_error
+          ].freeze
+
+          NOTICE_TITLE_MAPPING = {
```

Comment:
Let's move this constant to the `Application` model so that all notices related to the application are centralized within the `Application` model.

Reply:
Constant for notices is related to person documents model. I feel it should be in person model or in this operation as we only use this mapping in current feature UI display. In other places we use like IVL_OEA and similar kind.

Reply:
Sure, fine. Let's move it to the documents model.

Reply:
I mean we will not use OEA , OEM or OEU keys anywhere else except redetermination UI. Other places we will use IVL_OEA, IVL_OEM or IVL_OEU.

Reply:
@vkghub That makes sense. So, let's do one of the below:
- Use IVL_OEA, IVL_OEM or IVL_OEU and move it to Documents model **(OR)**
- Leave the constant in the operation itself with a comment to be very explicit about the constant that it is only used for UI purposes and DB has different values or something similar.